cmd/admin-config-get.go

```diff
@@ -67,31 +81,43 @@ func (u configGetMessage) JSON() string {
 	return strings.NewReplacer(`\n`, "", `\t`, "").Replace(string(statusJSONBytes))
 }
 
-// checkAdminConfigGetSyntax - validate all the passed arguments
-func checkAdminConfigGetSyntax(ctx *cli.Context) {
-	if len(ctx.Args()) == 0 || len(ctx.Args()) > 2 {
+// checkAdminConfigGetSyntax - validates arguments
+func checkAdminConfigGetSyntax(ctx *cli.Context) string {
+	if len(ctx.Args()) == 0 {
 		cli.ShowCommandHelpAndExit(ctx, "get", 1) // last argument is exit code
 	}
+	if len(ctx.Args()) == 1 {
+		return "fullGet"
+	}
+	// if 2 or more arguments are passed
+	return "partialGet"
 }
 
 func mainAdminConfigGet(ctx *cli.Context) error {
+	// Initializations
+	aliasedURL := ctx.Args().Get(0)
+	isFullConfigGet := false
 
-	checkAdminConfigGetSyntax(ctx)
-
-	// Get the alias parameter from cli
-	args := ctx.Args()
-	aliasedURL := args.Get(0)
+	// Check command arguments
+	if checkAdminConfigGetSyntax(ctx) == "fullGet" {
+		isFullConfigGet = true
+	}
 
 	// Create a new Minio Admin Client
 	client, err := newAdminClient(aliasedURL)
 	fatalIf(err, "Cannot get a configured admin connection.")
-
-	// Call get config API
-	c, e := client.GetConfig()
-	fatalIf(probe.NewError(e), "Cannot get server configuration file.")
-
-	// Print
-	printMsg(configGetMessage{Config: string(c)})
-
+	if isFullConfigGet {
+		// Call get config API
+		c, e := client.GetConfig()
+		fatalIf(probe.NewError(e), "Cannot get server configuration file.")
+		printMsg(configGetMessage{Config: string(c)})
+	} else {
+		argsList := ctx.Args().Tail()
+		// Call get config keys API
+		c, e := client.GetConfigKeys(argsList)
+		fatalIf(probe.NewError(e), "Cannot get server configuration file.")
```

Comment:
The message has to reflect the fact that we are requesting some keys and not a whole file.

Reply:
OK